pkg/runtime/wfengine/workflow.go

```diff
@@ -290,6 +293,7 @@ func (wf *workflowActor) addWorkflowEvent(ctx context.Context, actorID string, h
 	return wf.saveInternalState(ctx, actorID, state)
 }
 
+// [Question] Does this invoke ? or schedules a reminder for the workflow ?
```

Comment:
`runWorkflow` schedules the workflow(https://github.com/dapr/dapr/blob/HEAD/pkg/runtime/wfengine/workflow.go#L378-L379). The reminder would have been already invoked, which is what calls this method.

Reply:
Oh, I get it now. Things are like this.

- `createReliableReminder` creates a Reminder.
- `invokeReminder` in Internal Actor Interface, is implemented by Activities and Workflow. 
- They call the `runWorkflow` and `executeActivity` methods, which are implementing the internal Actor interface.

Thanks for explaining this.